packages/contracts/src/core/dao/IDAO.sol

```diff
@@ -55,13 +55,15 @@ interface IDAO {
     /// @param actions The array of actions executed.
     /// @param failureMap The failure map encoding which actions have failed.
     /// @param execResults The array with the results of the executed actions.
+    /// @param allowFailureMap The encoded uint to mark which actions are allowed to fail.
     /// @dev The value of `callId` is defined by the component/contract calling the execute function. A `Plugin` implementation can use it, for example, as a nonce.
     event Executed(
         address indexed actor,
         bytes32 callId,
         Action[] actions,
         uint256 failureMap,
-        bytes[] execResults
+        bytes[] execResults,
+        uint256 allowFailureMap
```

Comment:
```suggestion
    /// @param failureMap The allow failure map encoding which actions are allowed to fail.
```
Is it possible to rearrange the order?

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        uint256 failureMap,
        bytes[] execResults
    );
```
or 

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        bytes[] execResults,
        uint256 failureMap
    );
```
seems more logical to me. (`allowFailureMap` as an input coming before `failureMap`, the output)